docs/Getting Started/Debian/Debian Bullseye Root on ZFS.rst

```diff
@@ -20,7 +20,7 @@ System Requirements
 ~~~~~~~~~~~~~~~~~~~
 
 - `64-bit Debian GNU/Linux Bullseye Live CD w/ GUI (e.g. gnome iso)
-  <https://cdimage.debian.org/mirror/cdimage/unofficial/non-free/cd-including-firmware/current-live/amd64/iso-hybrid/debian-live-11.0.0-amd64-standard+nonfree.iso>`__
+  <https://cdimage.debian.org/mirror/cdimage/release/current-live/amd64/iso-hybrid/>`__
```

Comment:
Please give a chance to root on ZFS to people with closed source non-free firmware network cards to run debootstrap ;)